### PR TITLE
fix: Remove Search menu viewing in toolbar in EventDetailsFragment

### LINKED ISSUE #231: Search view still visible in toolbar after clicking on any event in SearchFragment
screenshot_20180701-025549

Working on it :+1:

### PR DESCRIPTION
**Pull request by dreadpool2**
Fixes #231

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt [hunk 1]
```diff
@@ -238,6 +238,7 @@ class EventDetailsFragment : Fragment() {
     }
 
     override fun onPrepareOptionsMenu(menu: Menu?) {
+        menu?.setGroupVisible(R.id.searchMenu, false)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt]: snake case

**Comment by dreadpool2** [comment]
PR Updated

File: app/src/main/res/menu/profile.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
             android:title="@string/logout" />
     </group>
 
-    <group android:id="@+id/searchMenu">
+    <group android:id="@+id/search_menu">
         <item
             android:id="@+id/searchItem"
```
Comment (srv-twry) [inline app/src/main/res/menu/profile.xml]: this one also.

File: app/src/main/res/menu/profile.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
-    <group android:id="@+id/profileMenu">
+    <group android:id="@+id/profile_menu">
         <item
             android:id="@+id/orgaApp"
```
Comment (srv-twry) [inline app/src/main/res/menu/profile.xml]: here

Comment (dreadpool2) [reply app/src/main/res/menu/profile.xml]: It was only for group ids I presume

Comment (dreadpool2) [reply app/src/main/res/menu/profile.xml]: @srv-twry @iamareebjamal What should be done

Comment (srv-twry) [reply app/src/main/res/menu/profile.xml]: https://stackoverflow.com/questions/35923922/best-practices-for-id-naming-conventions-in-android

Every id should be in snakecase.

Comment (dreadpool2) [reply app/src/main/res/menu/profile.xml]: But then we have always used camel case for previous ids!

Comment (srv-twry) [reply app/src/main/res/menu/profile.xml]: @iamareebjamal will have the final say. I am still in favour of snake case for everything in xml.

Comment (iamareebjamal) [reply app/src/main/res/menu/profile.xml]: For the ones you are referring in kotlinx should remain in camel case, i.e., view IDs. Others should be in snake case

Comment (srv-twry) [reply app/src/main/res/menu/profile.xml]: I was informed on another PR that "view ids should remain to be camel case as they are picked up by kotlinx".

**Pull request merged**